pytubefix/contrib/search.py

```diff
@@ -14,7 +14,8 @@
 class Search:
     def __init__(
             self, query: str,
-            client: str = InnerTube().client_name,
+            params: Optional[Dict[str, str]] = None,
+            client: str = 'WEB',
```

Comment:
The WEB client should **only be used for pagination**. The YouTube, Playlist and Channel classes must receive the default or custom client.